index.html

```diff
@@ -1,39 +1,121 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PROJECT ENVATO TEAM FURY CREATIVE PORTFOLIO</title>
+    <!-- bootstrap -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <!-- font awesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./assets/css/portfolio.css" />
+  </head>
 
-<head>
-	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>PROJECT ENVATO TEAM FURY CREATIVE PORTFOLIO</title>
-	<link rel="stylesheet" href="./assets/css/portfolio.css">
-	<script src="./assets/js/portfolio.js" async></script>
-</head>
+  <body>
+    <main>
+      <section class="introduction">
+        <header class="header">
+          <div class="container">
+            <nav class="navbar navbar-expand-md navbar-light">
+              <!-- Brand -->
+              <a class="navbar-brand" href="#">Axis</a>
 
-<body>
-	<main>
-		<section class="introduction">
-			<header>
-				<h1>PROJECT STRUCTURE</h1>
-			</header>
-		</section>
+              <!-- Toggler/collapsibe Button -->
+              <button
+                class="navbar-toggler"
+                type="button"
+                data-toggle="collapse"
+                data-target="#collapsibleNavbar"
+              >
+                <span class="navbar-toggler-icon"></span>
+              </button>
 
-		<section class="skills">
-		</section>
+              <!-- Navbar links -->
+              <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav ml-auto">
+                  <li class="nav-item active">
```

Comment:
The active link highlight does not look like the design. Please take another look